wrappers/nodejs/src/addon.cpp

```diff
@@ -1803,7 +2032,7 @@ class RSDevice : public Nan::ObjectWrap {
     if (me) {
       int32_t on = rs2_supports_device_info(me->dev,
           (rs2_camera_info)camera_info, &me->error);
-      info.GetReturnValue().Set(Nan::New(on));
+      info.GetReturnValue().Set(Nan::New(on?true:false));
```

Comment:
Can not understand why this change? If needed, please use space around `?` and `:`

Reply:
This change is to ensure the return value type is a boolean instead of number. I will add the missing spcaes.